modules/admin/proc-migrating-databases-to-an-external-server.adoc

```diff
@@ -0,0 +1,92 @@
+[id="proc-migrating-databases-to-an-external-server_{context}"]
+= Migrating local databases to an external database server
+
+By default, {product} hosts the data for each plugin in a PostgreSQL database. When you fetch the list of databases, you might see multiple databases based on the number of plugins configured in {product-short}. You can migrate the data from an {product-very-short} instance hosted on a local PostgreSQL server to an external PostgreSQL service, such as AWS RDS, Azure database, or Crunchy database. To migrate the data from each {product-very-short} instance, you can use PostgreSQL utilities, such as link:https://www.postgresql.org/docs/current/app-pgdump.html[`pg_dump`] with link:https://www.postgresql.org/docs/current/app-psql.html[`psql`] or link:https://www.pgadmin.org/[`pgAdmin`]. 
+
+[NOTE]
+====
+For quick migration, you can also use the link:https://github.com/janus-idp/operator/blob/main/hack/db_copy.sh[database copy script].
+====
+
+.Prerequisites
+
+* You have installed the link:https://www.postgresql.org/docs/current/app-pgdump.html[`pg_dump`] and link:https://www.postgresql.org/docs/current/app-psql.html[`psql`] utilities on your local machine.
+* For data export, you have the PGSQL user privileges to make a full dump of local databases.
+* For data import, you have the PGSQL admin privileges to create an external database and populate it with database dumps.
+
+.Procedure
+
+. Configure port forwarding for the local PostgreSQL database pod by running the following command on a terminal: 
++
+[source,terminal]
+----
+oc port-forward -n <your-namespace> <pgsql-pod-name> <forward-to-port>:<forward-from-port>
+----
+Where:
+* The `<pgsql-pod-name>` variable denotes the name of a PostgreSQL pod with the format `backstage-psql--<_index>`.
+* The `<forward-to-port>` variable denotes the port of your choice to forward PostgreSQL data to.
+* The `<forward-from-port>` variable denotes the local PostgreSQL instance port, such as `5432`.
++
+.Example: Configuring port forwarding
+[source,terminal]
+----
+oc port-forward -n backstage backstage-psql-backstage1-0 15432:5432
+----
+
+. Make a copy of the `db_copy.sh` script and edit the following details based on your configuration:
```

Comment:
This guide, as it is, relies on using the `db_copy.sh`. If we don't want to go general (and just say copy all the `allDB` databases at this point), we should make the `db_copy.sh` part of this guide (not just put it as a note). Also, we have to at least point to the right branch according to the version of RHDH (e.g. https://github.com/janus-idp/operator/blob/1.2.x/hack/db_copy.sh for 1.2). Or we can put this script into the documentation itself.

Reply:
added the script in the doc itself for better clarity.